components/02-form-elements/mutually-exclusive/mutually-exclusive.js

```diff
@@ -13,10 +13,7 @@ export default function mutuallyExclusiveInputs() {
     const checkboxElement = exclusiveWrapperElement.getElementsByClassName(checkboxClass)[0];
     const voiceOverAlertElement = exclusiveWrapperElement.getElementsByClassName(voiceOverAlertClass)[0];
     for (let exclusiveGroupElement of exclusiveGroupElements) {
-      const elementType = exclusiveGroupElement.type;
-      let event = elementType === 'checkbox' ? event = 'change' : event = 'keydown';
-
-      exclusiveGroupElement.addEventListener(event, function() {
+      exclusiveGroupElement.addEventListener('input', function() {
         voiceOverAlertElement.innerHTML = '';
         inputToggle(checkboxElement, voiceOverAlertElement, 'checkbox');
```

Comment:
`elType` is hardcoded here so it appears the `elType === 'text' || elType === 'textarea'` and `elType === 'select-one'` will never run

Reply:
Yes that's correct. We only want the checkbox type to run as that's only ever going to be a checkbox (the override).

Reply:
Ah i understand... maybe some comments might around the function might be helpful